pom.xml

```diff
@@ -123,6 +123,12 @@
 	    <version>5.8.1</version>
 	    <scope>test</scope>
 	</dependency>
+	<dependency>
+		<groupId>junit</groupId>
+		<artifactId>junit</artifactId>
+		<version>4.13.1</version>
+		<scope>test</scope>
+	</dependency>
```

Comment:
I suspect that what is happening is that Eclipse might add JUnit to the class path automatically.  Dunno, but it seems to be missing when I run it.